Server/mods/deathmatch/logic/luadefs/CLuaACLDefs.h

```diff
@@ -48,4 +48,7 @@ class CLuaACLDefs : public CLuaDefs
     LUA_DECLARE(aclGroupRemoveObject);
     LUA_DECLARE_OOP(isObjectInACLGroup);
     LUA_DECLARE(hasObjectPermissionTo);
+
+    static std::variant<std::vector<CAccessControlListGroup*>, bool>
+        aclObjectGetGroups(std::string strObject);
```

Comment:
This function returns a vector on every return path now, get rid of the variant.